### PR TITLE
This contact-branch has layout for the form

### PR DESCRIPTION
**Pull request by Solo7991**
Changed body margin to auto
Adjusted for viewports larger than 767px
Added desktop-class to cards

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -353,26 +356,18 @@ <h2>Contact me</h2>
...
                                         name="email">
                                 </label>
                                 <label for="more"><textarea 
-                                    id="more" 
-                                    required
-                                    maxlength="500"
+                                    id="more"  required maxlength="500"
                                     name="more"
                                     >Write your message here</textarea>
                                 </label>
```
Comment (roshan-bajgain) [inline index.html]: - [x] Kindly implement a placeholder for the text.

File: index.html [hunk 2]
```diff
@@ -353,26 +356,18 @@ <h2>Contact me</h2>
...
+                                        type="email" placeholder="jack32@gmail.com"
+                                        id="email" required
                                         name="email">
                                 </label>
                                 <label for="more"><textarea 
-                                    id="more" 
-                                    required
-                                    maxlength="500"
+                                    id="more"  required maxlength="500"
                                     name="more"
```
Comment (roshan-bajgain) [inline index.html]: - [x] Kindly stick as much as possible with Figma design. Your spacing is bigger in between buttons and text. so Kindly implement it, and implement the button border as shown in the Figma design.
### your design
your design
### Figma design
figma

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight :tada: :champagne: 
- Nice PR description and title :+1: 
- Nice README file :heavy_check_mark:
- No linter error :heavy_check_mark:
- GitHub flow is followed :+1:


## Required Changes ♻️
- [x] I would suggest you not merge the pull request until the final approval. Use the same pull request for required changes.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OmarRamoun — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @Solo7991 👋

Your project is complete! ✔️ 🏁

Finally, **you did it**! 🎉 👏. Your code was very professional 💯 and I can see you have a great chance to be a Great Software Engineer ⛑️.

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉. And Welcome to Microverse 👋.

> **Note:** You can now take a break even if short before you move on to the next project. Happy 😄 Coding 🖥️! and Have a nice day 🌞.

## Highlights 🌟


- ✔️ descriptive commit messages
- ✔️ descriptive PR title and description
- ✔️ used GitHub flow
- ✔️ UI Design is so close to figma
- The code is top-notch! 💯 and have a huge potential.
Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️
- This is My **GitHub**: [@mrramoun](https://github.com/mrramoun)
- This is My **Twitter**: [@ramoun16](https://twitter.com/ramoun16)
- This is My **Slack**: [Ramoun](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**